check_nextcloud.php

```diff
@@ -6,7 +6,7 @@
  * Monitoring plugin to check the status of nextcloud serverinfo app
  *
  * Copyright (c) 2019 Kevin Köllmann <mail@kevinkoellmann.de>
- * contribution: Conrad Beckert <kontakt@miradata.de>
+ * contribution and further development: Conrad Beckert <kontakt@miradata.de>
```

Comment:
I'm still active on this project. Please remove this line.

Reply:
ok